blog/modules/ROOT/pages/class-as-a-function.adoc

```diff
@@ -1,3 +1,7 @@
+:date: 2024-08-13
+:author: Josh Basch
+:author-url: https://github.com/HT154
```

Comment:
Note: we can't substitute these attributes in the byline, because it breaks when we embed that byline into `index.adoc`.